### ISSUE TITLE
Standardize the search indexing after sign-up

### ISSUE THREAD

**Issue by pixdemo2** [KeypDemoCorp/awesome-ui#129]
Make sure to update the relevant fixtures and snapshots.

This was flagged during the last round of QA testing.

Accessibility should be considered when implementing this.
